src/frontend/src/handler/mod.rs

```diff
@@ -131,6 +132,12 @@ pub(super) async fn handle(session: Arc<SessionImpl>, stmt: Statement) -> Result
             }
             create_index::handle_create_index(context, name, table_name, columns).await
         }
+        // Ignore `StartTransaction` and `Abort` temporarily.Its not final implementation.
+        // 1. Fully support transaction is too hard and gives few benefits to us.
+        // 2. Some client e.g. psycopg2 will use this statement.
+        // TODO: Track issues #2595 #2541
+        Statement::StartTransaction { .. } => Ok(PgResponse::empty_result(START_TRANSACTION)),
```

Comment:
lgtm, but it will be better to use `empty_result_with_notice` and give users a warning.

Reply:
Updated~  It will return below
```
Ignored temporarily.See detail in issue#2541
START_TRANSACTION
```